core/CMakeLists.txt

```diff
@@ -140,9 +140,6 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror -Wno-unknown-pragmas")
 if ($ENV{CURRENCY} MATCHES "btc-testnet")
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DBTC_TESTNET")
 elseif ($ENV{CURRENCY} MATCHES "btc-mainnet")
-  if (ENABLE_ASAN OR ENABLE_UBSAN)
-    message(FATAL_ERROR "Sanitizers are not supported with CURRENCY=btc-mainnet")
```

Comment:
It would make build_core.sh fail. There's actually no reason to prevent BTC_MAINNET from being built with sanitizers. TARGET=nCipher will do that independently because the cross-compiler doesn't support them.